packages/forms-web-app/src/controllers/full-planning/granted-or-refused.js

```diff
@@ -14,7 +14,7 @@ const {
 exports.forwardPage = (status) => {
   const statuses = {
     [applicationStatus.GRANTED]: '/before-you-start/decision-date',
-    [applicationStatus.NODECISION]: '/before-you-start/decision-date-due',
+    [applicationStatus.NODECISION]: '/before-you-start/date-decision-due',
```

Comment:
Just checking this isn't the decision date page, as that url is different: https://pins-ds.atlassian.net/browse/AS-3673?atlOrigin=eyJpIjoiOWRmODU5NmE3Y2Y0NGU3Njk1YzI5NmJiYWNhY2Q4MjEiLCJwIjoiaiJ9

Reply:
No, this is not _decision date_ page that you worked on